common/src/main/java/com/viaversion/viaversion/protocols/v1_20_3to1_20_5/Protocol1_20_3To1_20_5.java

```diff
@@ -188,7 +169,16 @@ protected void registerPackets() {
                 }
             }
 
-            replaceChatAck(wrapper, storage);
+            fixChatAck(wrapper, storage);
+        });
+        registerServerbound(ServerboundPackets1_20_5.CHAT_ACK, ServerboundPackets1_20_3.CHAT_ACK, wrapper -> {
```

Comment:
```suggestion
        registerServerbound(ServerboundPackets1_20_5.CHAT_ACK, wrapper -> {
```
If the name is the same, they can get automatically mapped

Reply:
Neat, thanks! :)